demo/time-picker-basic-demos.html

```diff
@@ -19,9 +19,38 @@ <h3>Value Resolution</h3>
       The <code>step</code> property specified in seconds affects the resolution of the time picker and visibility of the dropdown.
       If step is less than 15 mins dropdown is hidden
     </p>
-    <vaadin-demo-snippet id="time-picker-basic-demos-step-example">
+    <vaadin-demo-snippet id="time-picker-basic-demos-step-example" when-defined="vaadin-time-picker">
       <template preserve-content>
-        <vaadin-time-picker step="0.5" label="Duration (hh:mm:ss.fff)"></vaadin-time-picker>
+        <vaadin-combo-box id="combobox-step-selection" value="1" label="Step" style="width: 250px;"></vaadin-combo-box>
+        <vaadin-time-picker id="time-picker-basic-demos-value-resolution" step="1" label="Duration (hh:mm:ss.fff)"></vaadin-time-picker>
```

Comment:
Now the initial `label` doesn't match the initial `step`. With a `step` of 1 the initial label should be `Duration (hh:mm:ss)` instead of `Duration (hh:mm:ss.fff)`, however I think the initial/default step would be better to be e.g. 30 minutes so that you can see how it affects the dropdown without changing the step.

It would also be better to have a default value set (e.g. `12:00:00`) so that you can immediately see how changing the step affects the value format without you having to first set some value manually.

Reply:
Yes, indeed. Done.

P.S. thank you for the review!